packages/components/src/overlays/Popover/src/Popover.module.css

```diff
@@ -27,6 +28,10 @@
     --width: var(--hop-Popover-dropdown-inline-width);
 }
 
+.hop-Popover[data-trigger="ComboBox"] {
```

Comment:
I'm not sure the popover should be aware of the existence of Combobox.

Reply:
this is actually something added by RAC and they documented it.

Reply:
maybe we should go about ti differently, but I'm not entirely sure what the best way would be..

Reply:
If RAC does it. We used to add a class like hop-Combobox-Popover-trigger and style it within Combobox

Reply:
ya I thought about having it in combobox, but things could get complicated.. because a width is set directly on the popover. I guess we can be specific and make sure we override it in combobox. But i also remember you saying that if we're overriding inside another component, then maybe something is wrong. so I try not to do that where I can.